.github/workflows/lint_test.yml

```diff
@@ -0,0 +1,28 @@
+name: Linting and testing
+
+on:
+  - pull_request
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Run eslint
+        run: npm run lint
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Run jest
+        run: npm run test
```

Comment:
move the `run test` task above, we don't need a separate checkout and `npm i` (remove l18-26)

Reply:
also, it's `npm test`